gwakeonlan/import_ethers.py

```diff
@@ -0,0 +1,40 @@
+# @license AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```

Comment:
GPLv2 and AGPLv3 should be compatible AFAIK but I can change it to GPLv2 if you want?